README.md

```diff
@@ -162,6 +162,7 @@ class Example extends React.Component {
 | [`onSuggestionsClearRequested`](#on-suggestions-clear-requested-prop)  | Function | ✓[\*](#on-suggestions-clear-requested-prop-note) | Will be called every time you need to set `suggestions` to `[]`.                                                                                                                                      |
 | [`getSuggestionValue`](#get-suggestion-value-prop)                     | Function |                        ✓                         | Implement it to teach Autosuggest what should be the input value when suggestion is clicked.                                                                                                          |
 | [`renderSuggestion`](#render-suggestion-prop)                          | Function |                        ✓                         | Use your imagination to define how suggestions are rendered.                                                                                                                                          |
+| [`containerProps`](#container-props-prop) | Object | | Pass through arbitrary props to the container. Useful if you need to override the default props set by Autowhatever, for example, for accessibility. |
```

Comment:
Optional props, like this, should come after the required props in the table somewhere. This should be _below_ `inputProps`, so that it's easy for people to find all the required props when initially getting started.